pkg/apis/eksctl.io/v1alpha5/nodegroups.go

```diff
@@ -44,14 +46,28 @@ func ClusterHasInstanceType(cfg *ClusterConfig, hasType func(string) bool) bool
 	return false
 }
 
-// HasNodegroup returns true if this clusterConfig contains a managed or un-managed nodegroup with the given name
-func (c *ClusterConfig) FindNodegroup(name string) *NodeGroup {
+// FindNodegroup returns true if this clusterConfig contains a managed or un-managed nodegroup with the given name
```

Comment:
This comment needs to be updated to match the signature.

Reply:
thanks, updated it!